templates/ctl-scaffold/test/Main.purs

```diff
@@ -6,22 +6,32 @@ import Contract.Prelude
 
 import Contract.Config as Contract.Config
 import Contract.Monad as Contract.Monad
+import Contract.Test.Plutip (PlutipTest, testPlutipContracts)
 import Contract.Test.Plutip as Contract.Test.Plutip
-import Contract.Wallet as Contract.Wallet
+import Ctl.Internal.Test.TestPlanM (TestPlanM, interpretWithConfig)
```

Comment:
I thought I'd take a quick look before it's ready for review - we don't export interpretWithConfig, so we really shouldn't be using it in the template. We could export it, but ideally the way CTL internally tests and the way users test should both be able to change freely - if we export it, we have promises to keep.

I think we should copy https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/src/Internal/Test/TestPlanM.purs as `ctl-scaffold/test/Plan.purs` and drop the `TestPlanM` alias in the copy. I'd be tempted to not use our custom version of `consoleReporter` either, and use the built in one from `purescript-spec`.

Reply:
Thanks for the tip. as of now, we only need `interpretWithConfig`, but I left `interpret` and `interpretWithTimeout` in Plan.purs in case users want to use it. or should I remove it?

Reply:
No, we don't want to copy it. We want to make it a part of our API. Move `interpretWithConfig` and the like to `Contract.Test.Mote`. Our `consoleReporter` should go to `Contract.Test.Mote.ConsoleReporter` and should be used by default (it is better).

`interpretWithTimeout`  is not needed at all. I think we should drop it entirely, since it just sets a single config field.